### PR TITLE
[AIRFLOW-XXX] Dump logs in case of kube failure

### PR DESCRIPTION
**Pull request by ashb**
### Jira
- [x] None

### Description

- [x] Previously we were only dumping the logs in case of _success_, which was somewhat pointless, and made errors hard to track down.

### CODE REVIEW

File: scripts/ci/kubernetes/kube/deploy.sh [hunk 1]
```diff
@@ -199,21 +219,7 @@ if [[ "$AIRFLOW_WEBSERVER_IS_READY" == 1 ]]; then
   echo "Airflow webserver is ready."
 else
   echo "Airflow webserver is not ready after waiting for a long time. Exiting..."
-  exit 1
```
Comment (kaxil) [inline scripts/ci/kubernetes/kube/deploy.sh]: We still need `exit 1`, don't we ?

Comment (potiuk) [reply scripts/ci/kubernetes/kube/deploy.sh]: Yeah.

Comment (ashb) [reply scripts/ci/kubernetes/kube/deploy.sh]: Whoops yes

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
It will be really useful !

**Comment by ashb** [comment]
I'm testing a few more things - the second time we run the deploy in the script I'm not getting logs and I'm not sure why.

**Comment by ashb** [comment]
Sorted the problem - I was trying to find the `POD` to early so hitting a race condition. Will double check the logs after this build completes and merge if it looks good.

**Comment by ashb** [comment]
Travis is being a 🐌 to build our  PRs again. My fork passed https://travis-ci.com/ashb/airflow/jobs/211087975 so I'm merging it.

**Pull request merged**